markdown/community-guidelines.md

```diff
@@ -11,28 +11,45 @@ How to behave on Kitsu
 
 #### Privacy Violations
 
 Do not post your own or other people's private information, which includes but is not limited to real names, addresses, phone numbers, email addresses, and nonconsensual sexualized images or media (often referred to as “revenge porn”). We are against doxxing regardless of how awful you may think a person is. It can lead to dangerous real life situations and unnecessary legal consequences. Doxxing is a form of harassment, and we're not cool with it.
 
 #### Malicious Speech
 
+Don't encourage violence or hatred on the basis of things like race, ethnic origin, religion, disability, gender, age, veteran status, sexual orientation, or any other characteristics. Refrain from the use of slurs or otherwise offensive remarks meant to target users or groups based on any of the above mentioned characteristics. 
 
-Don't encourage violence or hatred on the basis of things like race, ethnic origin, religion, disability, gender, age, veteran status, sexual orientation, or any other characteristics. We encourage you to dismantle negative speech through discussion rather than censorship.
+We want honest and open discussion on Kitsu and for that to happen, we encourage you to behave in such a way that others feel welcome and secure in the community. You're free to express your opinion; however, it crosses the line into malicious speech when it, directly or indirectly, incites or propagates violence, hatred, or makes people fear that  you or others will act against them in a violent manner. We reserve the right to flush anything overtly malicious right down our digital toilet.
 
-You're still welcome to express your opinions, even if they are offensive to some users. We want honest and open discussion to happen on Kitsu from all sides. However, these opinions cross the line into malicious speech when they specifically incite violence or hatred, or make people fear you will act against them in a violent manner. We reserve the right to flush anything overtly malicious right down our digital toilet.
+#### Disinformation, Misinformation, and Propaganda.
+
+Don’t post or repost content that is verifiably false or misleading. Sharing content that is intended to intentionally deceive or may cause public harm is prohibited. Content that may cause public harm includes but is not limited to threats to democracy, health, and other topics which may impact a person or group negatively.
+
+Recognise that content and ideas circulated on online platforms, including hate, racial prejudice and disinformation, can have real and concrete consequences in the real world. This can disproportionately affect equity-deserving communities, including but not limited to, disabled people, neurodiverse, indigenous people, racialized communities, religious minorities, gender diverse, and members of LGBTQIA+ communities.
```

Comment:
```suggestion
Recognize that content and ideas circulated on online platforms — including hate, prejudice and disinformation — can have real and concrete consequences in the real world. This can disproportionately affect marginalized groups, including but not limited to: disabled, neurodiverse, indigenous, gender-diverse, or LGBTQIA+ individuals, as well as members of racial and religious minorities.
```